### ISSUE TITLE
Cyrillic symbols support

### ISSUE THREAD

**Issue by AiratHalitov** [AffiliateWP/AffiliateWP#1771]
AffiliateWP 1.9.4 don't support cyrillic symbols in "Export to CSV" function.

**Comment by AiratHalitov** [comment]
I found.
It looks normal in notepad++ (UTF-8), but in Excel (Windows-1251) it looks like this
_"Р•РјРµР№Р»","РЎСѓРјРјР°","Р’Р°Р»СЋС‚Р°"_

**Comment by pippinsplugins** [comment]
@tubiz could you work on this please?

**Comment by tubiz** [comment]
@pippinsplugins will do.

**Comment by tubiz** [comment]
PR: #2617 

#### How to replicate

* Modify any referral and set the referral description to Russian (Pусский)
* Set the site language to Russian (Pусский). 
* Export any data (affiliates, referrals, payouts, visits) from the site.
* Open the exported file and set the character set to Cyrillic (Windows-1251). 
* The csv header and the referral description will be displayed in an invalid format.

#### To Fix

* Before exporting data the customer will have to set the character encoding using a constant `AFFILIATE_WP_EXPORT_CHARSET` in the wp-config file.

##### Example
`define( 'AFFILIATE_WP_EXPORT_CHARSET', 'Windows-1251' );`

This character encoding will be used when exporting data.

#### Note
A documentation will have to be created for this in the docs.

#### New filter `affwp_referral_export_get_data_line` added
A new filter was added `affwp_referral_export_get_data_line` to be able to change the character encoding when exporting referrals.

**Comment by pippinsplugins** [comment]
@amdrew could you please test this?

**Comment by amdrew** [comment]
Hopefully I have tested correctly (I'm on a mac). 

First, I opened my exported file in textedit and noted I could view the Russian characters just fine. TextEdit uses `Automatic` as the default setting for the `Opening Files` option:

<img width="961" alt="screen shot 2018-03-29 at 12 31 52 pm" src="https://user-images.githubusercontent.com/52581/38061775-2dc1f59c-334d-11e8-84cd-955f83e94b2d.png">


Then to replicate the issue I changed TextEdit's encoding to `Cyrillic (Windows)` when opening files.

<img width="551" alt="screen shot 2018-03-29 at 12 25 29 pm" src="https://user-images.githubusercontent.com/52581/38061624-651be288-334c-11e8-947e-8a103673a2fa.png">

After doing this, I was unable to open my exported .csv file in TextEdit:

<img width="532" alt="screen shot 2018-03-29 at 12 25 05 pm" src="https://user-images.githubusercontent.com/52581/38061625-654890c6-334c-11e8-9820-a88e5c632c7c.png">

After switching to the proposed PR and adding the constant, then exporting my file again, I was able to open it in textedit with `Cyrillic (Windows)` still enabled.

<img width="905" alt="screen shot 2018-03-29 at 12 25 55 pm" src="https://user-images.githubusercontent.com/52581/38061623-64f00aa0-334c-11e8-93ee-205cb92a8d8c.png">
